.github/workflows/apply-stale-closure-label.yml

```diff
@@ -0,0 +1,33 @@
+name: Add [Closed] Stale label to closed issues
+
+on:
+  issues:
+    types: closed
+
+jobs:
+  add_label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Get comments
+        uses: octokit/request-action@v2.x
+        id: get_comments
+        with:
+          route: GET /repos/${{github.repository}}/issues/${{ github.event.issue.number }}/comments
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set Last Comment
+        id: get_last_comment
+        run: |
+          last_comment=$( echo '${{ steps.get_all_comments.outputs.data }}' | tr -d '[:cntrl:]' | jq -r '.[-1].body' )
```

Comment:
_(I'm not too familiar with the workflow.)_

>steps.get_all_comments.outputs.data 

**question:** The ID used in the previous step was `get_comments`. But, `get_all_comments` is being referenced here. Shouldn't the reference be `${{ steps.get_comments.outputs.data }}`?

Reply:
Good catch, thank you for your eagle-eyed observation!